schemas/mozza/edge-validator/edge-validator.1.parquetmr.txt

```diff
@@ -0,0 +1,14 @@
+message edge-validator {
+    required group results {
+        required int32 error_count;
+        required float32 error_rate;
+        opetional group errors (MAP) {
```

Comment:
typo

Reply:
I think this typo is what's causing the tests to fail.